web/src/components/storage/MenuDeviceDescription.tsx

```diff
@@ -38,13 +42,13 @@ export default function MenuDeviceDescription({ device }: { device: StorageDevic
         <span>{typeDescription(device)}</span>
         <span>{contentDescription(device)}</span>
       </Split>
-      <Split hasGutter>
-        {device.systems.map((s, i) => (
-          <Label key={i} isCompact>
+      <Flex columnGap={{ default: "columnGapXs" }}>
+        {filesystemLabels(device).map((s, i) => (
+          <Label key={i} variant="outline" isCompact>
             {s}
           </Label>
```

Comment:
If I'm not wrong I have seen this code before. Something to extract if we finally took some time for refactoring DriveEditor and friends.

Reply:
Not now, but definitely in the mid-term that's absolutely needed.